experimental/dds/tree2/src/feature-libraries/forestSummarizer.ts

```diff
@@ -110,3 +137,19 @@ export class ForestSummarizer implements Summarizable {
 		}
 	}
 }
+
+function encodeSummary(
+	cursor: ITreeCursorSynchronous,
+	schema: StoredSchemaCollection,
+	policy: FullSchemaPolicy,
+	encodeType: TreeCompressionStrategy,
+): EncodedChunk {
+	switch (encodeType) {
+		case TreeCompressionStrategy.Compressed:
+			return schemaCompressedEncode(schema, policy, cursor);
+		case TreeCompressionStrategy.Uncompressed:
+			return uncompressedEncode(cursor);
+		default:
+			return schemaCompressedEncode(schema, policy, cursor);
```

Comment:
I think using unreachableCase here would make this code less likley to get out of date if we added more options.

Reply:
Updated default case to unreachableCase